src/viur/core/bones/selectcountry.py

```diff
@@ -765,32 +22,778 @@ class SelectCountryBone(SelectBone):
+        'bhr': 'bh',
+        'pry': 'py',
+        'sau': 'sa',
+        'cze': 'cz',
+        'qat': 'qa',
+        'ukr': 'ua',
+        'cym': 'ky',
+        'afg': 'af',
+        'bgr': 'bg',
+        'vgb': 'vg',
+        'nam': 'na',
+        'grd': 'gd',
+        'grc': 'gr',
+        'twn': 'tw',
+        'khm': 'kh',
+        'grl': 'gl',
+        'pak': 'pk',
+        'srb': 'rs',
+        'pan': 'pa',
+        'syc': 'sc',
+        'npl': 'np',
+        'kir': 'ki',
+        'mar': 'ma',
+        'lbn': 'lb',
+        'phl': 'ph',
+        'nic': 'ni',
+        'vnm': 'vn',
+        'iot': 'io',
+        'syr': 'sy',
+        'mac': 'mo',
+        'maf': 'mf',
+        'kaz': 'kz',
+        'cok': 'ck',
+        'pyf': 'pf',
+        'niu': 'nu',
+        'svn': 'si',
+        'egy': 'eg',
+        'svk': 'sk',
+        'dom': 'do',
+        'mrt': 'mr',
+        'esp': 'es',
+        'fsm': 'fm',
+        'kna': 'kn',
+        'gha': 'gh',
+        'cck': 'cc',
+        'chl': 'cl',
+        'ury': 'uy',
+        'tha': 'th'
+    }
+    """A Map of ISO3 to ISO2 country codes"""
+
+    ISO2TOISO3 = {v: k for k, v in ISO3TOISO2.items()}  # Build the invert map
+    """A built Map of ISO2 to ISO3 country codes"""
+
+    subgroup_mappings = {
+        'dach': ['de', 'at', 'ch'],
+        'eu': ['at', 'be', 'bg', 'hr', 'cy', 'cz', 'dk', 'ee', 'fi', 'fr', 'de', 'gr', 'hu', 'ie', 'it', 'lv', 'lt',
+               'lu',
+               'mt', 'nl', 'pl', 'pt', 'ro', 'sk', 'si', 'es', 'se'],
+    }
+
     type = "select.country"
     ISO2 = 2
     ISO3 = 3
 
-    def __init__(self, *, codes=ISO2, values=None, **kwargs):
-        global ISO2CODES, ISO3CODES
+    def __init__(self, *, descr="Country", codes=ISO2, values=None, **kwargs):
         assert codes in [self.ISO2, self.ISO3]
         # assert values is None
 
         if values is None:
-            values = OrderedDict(sorted((ISO2CODES if codes == self.ISO2 else ISO3CODES).items(), key=lambda i: i[1]))
+            values = OrderedDict(
+                sorted((self.ISO2CODES if codes == self.ISO2 else self.ISO3CODES).items(), key=lambda i: i[1]))
         else:
             if isinstance(values, str):
-                values = subgroup_mappings[values]
+                values = self.subgroup_mappings[values]
                 if codes == self.ISO3:
-                    values = [ISO2TOISO3[i] for i in values]
+                    values = [self.ISO2TOISO3[i] for i in values]
             if isinstance(values, list):
                 if codes == self.ISO2:
-                    values = {k: ISO2CODES[k] for k in values}
+                    values = {k: self.ISO2CODES[k] for k in values}
                 elif codes == self.ISO3:
-                    values = {k: ISO3CODES[k] for k in values}
+                    values = {k: self.ISO3CODES[k] for k in values}
             if isinstance(values, dict):
                 values = OrderedDict(sorted(values.items(), key=lambda i: i[1]))
```

Comment:
OrderedDict is no longer required here, please update this part as well.
```suggestion
                values = dict(sorted(values.items(), key=lambda item: item[1])))
```